Assets/Scripts/EnemyScripts/EnemyHealth.cs

```diff
@@ -17,45 +18,47 @@ public class EnemyHealth : MonoBehaviour
 
     void Update()
     {
-       
+
         if (health <= 0)
         {
             SFXManager.Instance.EnemyDieSound();
             ScoreManager.Instance.IncreasePoints(10);
             EnemySpawner.currentEnemies.Remove(gameObject);
             Destroy(gameObject);
-        } 
-        if (GameSettings.gameState != GameState.InGame){return;}
+        }
+        if (GameSettings.gameState != GameState.InGame) { return; }
         Bleed();
     }
     void Bleed()
     {
-        if (!bleeding || WeaponStats.Instance.BleedDamage == 0){return;} //If the enemy is not bleeding, return. This means there is a 1 second interval before the first bleed tick
-        bleedTick -= Time.fixedDeltaTime;
+        if (!bleeding || WeaponStats.Instance.BleedDamage == 0) { return; } //If the enemy is not bleeding, return. This means there is a 1 second interval before the first bleed tick
+        bleedTick -= Time.deltaTime;
         if (bleedTick <= 0)
         {
             bleedTick = BLEED_INTERVAL;
-            ReceiveDamage((baseHealth * WeaponStats.Instance.BleedDamage) / 100, false);
+            ReceiveDamage(Math.Max((baseHealth * WeaponStats.Instance.BleedDamage) / 100, 1), false);
```

Comment:
This bleed doesn't quite work as intended, as baseHealth is the health amount without the per-wave stat scaling. This means as enemy health goes up each round bleed damage stays the same. This will be fixed by my branch so no changes needed here.